client-js/app/test/index.html

```diff
@@ -1,22 +0,0 @@
-<!doctype html>
```

Comment:
I don't know how this got into master, but I don't think we want to use web-component tester anyhow...